src/RewriteRules.php

```diff
@@ -41,4 +41,23 @@ public static function namespaceRewrite()
             'ZendDiagnostics\\' => 'Laminas\\Diagnostics\\',
         );
     }
+
+    /**
+     * @return array
+     */
+    public static function classRewrite()
+    {
+        return array(
+            'Zend\\Expressive\\Authorization\\Acl\\ZendAclFactory'
+                => 'Laminas\\Expressive\\Authorization\\Acl\\LaminasAclFactory',
```

Comment:
Except this should be `Expressive\Authorization\Acl\LaminasAclFactory` (should not have the `Laminas` top-level namespace).

I'll fix on merge.

Reply:
@weierophinney good catch, thanks !

Reply:
I've updated it now, thanks !